packages/solid/src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,278 @@
+import { createStitches, createTheme } from "@stitches/core";
+import merge from "just-merge";
+import {
+  Auth as AuthProps,
+  Localization,
+  I18nVariables,
+  SocialLayouts,
+} from "../../types";
+import { VIEWS } from "../../constants";
+import {
+  EmailAuth,
+  EmailAuthProps,
+  ForgottenPassword,
+  MagicLink,
+  SocialAuth,
+  UpdatePassword,
+} from "./interfaces";
+import { UserContextProvider, useUser } from "./UserContext";
+
+import * as _defaultLocalization from "../../../common/lib/Localization";
```

Comment:
This path isn't resolving as the directory name is lowercase (`localization`) while you are referencing capital case here (`Localization`).